🚧Update org contribution doc (#47)
KinsonDigital/.github: docs/EarlyPullRequests.md

<h2 style="border:0;font-weight:bold" align="center">Creating Early Pull Request</h2>

1. If you have not already done so, complete the **Setting Things Up** section above.
2. If you have not already done so, complete the **Create a Branch to Contribute** section above.
3. Create an empty commit.  This allows you to create a PR without any changes.
   ```cli
   git commit --allow-empty -m "Start work for issue #123"
   git push
   ```
4. Create a [**draft pull request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests). Do this **BEFORE** you start working.
   > **Note**

>    Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue.
>    > - Use one of the PR templates below depending on if you are doing a regular feature or a preview feature.

Reviewer: ```suggestion
   Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue (emoji's included).
   > - Use one of the PR templates below depending whether you are doing a regular feature or a preview feature.
```

Author: I do not want to call out the emoji thing since the templates already do.  If for some reason in the future, we decide to get rid of emojis, I don't want to have to update this document.  Copy the title example means the emojis for the most part anyways.

4. Create a [**draft pull request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests). Do this **BEFORE** you start working.
   > **Note**
   Please make sure that the title of the PR _**EXACTLY**_ matches the title of the associated issue.

>    > - Use one of the PR templates below depending on whether if you are doing a regular feature or a preview feature.

Reviewer: Remove 'if'
```suggestion
   > - Use one of the PR templates below depending on whether you are doing a regular feature or a preview feature.
```
